src/sysext/sysext.c

```diff
@@ -1049,6 +1134,10 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_noexec = r;
                         break;
 
+                case ARG_NO_RELOAD:
+                        arg_no_reload = true;
```

Comment:
This is still default on - it needs to be the other way around, default off, as it is a (likely broken) workaround

Reply:
I disagree. This is opt-in in the unit files, since the extension-release setting defaults to off there. I don't think we need a second layer of opt-in.

Hence: i am very sure it should be opt-in on the extension level and opt-out here in the sysext tool

Reply:
The problem is that doing a reload during boot is likely to break stuff, so simply adding a sysext will cause that to happen automatically - so it would be better if it was fully opt-in so that both the system configurer and who writes the extensions have to opt in